sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -413,6 +363,38 @@ case class LoadDataCommand(
     CommandUtils.updateTableStats(sparkSession, targetTable)
     Seq.empty[Row]
   }
+
+  /**
+   * Returns a qualified path object. Method ported from org.apache.hadoop.fs.Path class.
+   *
+   * @param defaultUri default uri corresponding to the filesystem provided.
+   * @param workingDir the working directory for the particular child path wd-relative names.
+   * @param path       Path instance based on the path string specified by the user.
+   * @return qualified path object
+   */
+  private def makeQualified(defaultUri: URI, workingDir: Path, path: Path): Path = {
+    val pathUri = {
```

Comment:
We're close now, but these braces are still unndeeded. I also tend to advise flipping an "if" condition so it's not negated when the clauses are equally important.